core/store/tx_manager.go

```diff
@@ -555,12 +555,16 @@ func (txm *EthTxManager) CheckAttempt(txAttempt *models.TxAttempt, blockHeight u
 		return receipt, Unconfirmed, nil
 	}
 
+	// TODO: Could make this configurable i.e. by setting
+	// minOutgoingConfirmations param on the ethtx adapter and taking a max of
+	// that and the globally configured value as we do with
+	// MinRequiredIncomingConfirmations
```

Comment:
Could this TODO be solved as part of this PR?

Reply:
Maybe, reading this now it doesnt even make sense so at the very least needs rewording

Reply:
Yes, OK I remember now.

We can put a `minimumRequiredOutgoingConfirmations` parameter on either EthTX adapter or the Job Spec as a whole. Which is preferable? @j16r @se3000

Reply:
After a lot of faffing I've parked this work in another branch since it was more involved than expected. Card for it here: https://www.pivotaltracker.com/n/projects/2129823